### PR TITLE
DL-5562 TemplateRenderer/MainTemplate changes so far

### PR DESCRIPTION
**Pull request by JoshMiernik**
# JIRA Story or Github Number - descriptive title

**Bug fix / New feature** (delete as appropriate)

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## Checklist

 - [ ]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions
 - [ ]  I've added my code using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: app/controllers/LookupController.scala [hunk 1]
```diff
@@ -42,7 +42,11 @@ class LookupController @Inject()(val keyStoreConnector: KeyStoreConnector,
                                  val plaConnector: PLAConnector,
                                  val actionWithSessionId: ActionWithSessionId,
                                  mcc: MessagesControllerComponents,
-                                 http: DefaultHttpClient)(
+                                 psa_lookup_not_found_results: views.html.pages.lookup.psa_lookup_not_found_results,
```
Comment (jgrath) [inline app/controllers/LookupController.scala:45]: I would use camel case here, if you can change it that would be great.

File: app/views/pages/amends/amendIP14CurrentPensions.scala.html [hunk 2]
```diff
@@ -24,6 +24,9 @@
 @import play.api.Application
 @import config.FrontendAppConfig
 
+@this(
```
Comment (jgrath) [inline app/views/pages/amends/amendIP14CurrentPensions.scala.html:27]: Should be one liner...

File: conf/application.conf [hunk 3]
```diff
@@ -22,6 +22,21 @@ play.i18n.langs = ["en", "cy"]
 
 play.http.secret.key="yNhI04vHs9<_HWbC`]20u`37=NGLGYY5:0Tg5?y`W<NoJnXWqmjcgZBec@rOxb^G"
 
+play.filters.enabled  = [
```
Comment (jgrath) [inline conf/application.conf:25]: Do we need all these?

File: app/views/pages/amends/amendIP14CurrentPensions.scala.html (hunk 2 above)
Comment (JoshMiernik) [reply app/views/pages/amends/amendIP14CurrentPensions.scala.html:27]: Done

**Review by jgrath — APPROVED** [review]

**Pull request merged**